README.md

```diff
@@ -46,13 +46,13 @@ To check license headers in GitHub Actions, add a step in your GitHub workflow:
 
 ### Docker
 
-[Native Image](https://www.graalvm.org/22.3/reference-manual/native-image/) powered image (28MB):
```

Comment:
It grew to 90MB due to https://github.com/chainguard-images/images/issues/1952.

See https://github.com/korandoru/hawkeye/pull/102#issuecomment-1857327613.